cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -175,7 +181,9 @@ impl AuthorizationCall {
             None => None,
         };
 
-        let context = Context::from_json_value(self.context, schema.as_ref().map(|s| (s, &action)))
+        let context = serde_json::to_value(self.context)
+            .map_err(|e| [format!("Error encoding the context as JSON: {e}")])?;
```

Comment:
This change should be made to the `Context` constructor.

Reply:
are we serializing here or deserializing here?

Reply:
De-serialized to a `map` so that we can use an annotation to fail on duplicates. Before we pass back to Context constructor we have to serialize back to a json-value. If we want to forbid duplicates in the context contructor itself, than this api can't just take a json value.